src/MSAL.PCL/Internal/Cache/TokenCacheAccessor.cs

```diff
@@ -77,17 +77,17 @@ public IList<RefreshTokenCacheItem> GetRefreshTokens(TokenCacheKey tokenCacheKey
             return matchedRefreshTokens;
         }
 
-        public void DeleteToken(AccessTokenCacheItem accessToken‪Item)
+        public void DeleteAccessToken(AccessTokenCacheItem accessToken‪Item)
         {
-            TokenCachePlugin.DeleteToken(accessToken‪Item.GetTokenCacheKey());
+            TokenCachePlugin.DeleteAccessToken(accessToken‪Item.GetTokenCacheKey());
         }
 
         public void DeleteRefreshToken(RefreshTokenCacheItem refreshToken‪Item)
         {
             TokenCachePlugin.DeleteRefreshToken(refreshToken‪Item.GetTokenCacheKey());
         }
 
-        public IList<AccessTokenCacheItem> GetAllAccessTokens()
+        public IList<AccessTokenCacheItem> GetAllAccessTokens(string clientId)
         {
             ICollection<string> allTokensAsString = TokenCachePlugin.GetAllAccessTokens();
```

Comment:
@jasoncoolmax - I just realized that sharing cache between iOS native and Xamarin is a non-goal. I think I will go ahead and implement your feedback. If things change in future, we can revert it.